### PR TITLE
fix: broken tests and do some house keeping

### PR DESCRIPTION
**Pull request by millsp**
* Fixes broken tests
* Upgrades redwood
* Removes unused test

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -3,3 +3,7 @@ shared-workspace-lockfile=false
 
 # without this, installing one project installs all
 recursive-install=false
+
+# so client postinstall hook is not cached for all
+# https://github.com/pnpm/pnpm/releases/tag/v8.7.1
+side-effects-cache=false
```
Comment (millsp) [inline .npmrc:9]: Does not actually fix the issue because we have dependency caching, but works if there's no caching.

File: docker/almalinux-8-amd64-openssl-1.1.x/run.sh [hunk 2]
```diff
@@ -30,3 +30,4 @@ docker run -p 3000:3000 \
 sleep 15
 
 pnpm install
+pnpm prisma generate
```
Comment (millsp) [inline docker/almalinux-8-amd64-openssl-1.1.x/run.sh:33]: I rather add it here instead of having a custom project exclusion.

File: platforms-serverless/lambda/test.ts [hunk 3]
```diff
@@ -22,9 +22,9 @@ async function main() {
   const actual = JSON.stringify(original)
   console.log('actual', actual)
   // TODO Update to only expect on engine file after zip script was updated
-  let files = `,"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","libquery_engine-debian-openssl-1.1.x.so.node","libquery_engine-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]`
```
Comment (millsp) [inline platforms-serverless/lambda/test.ts:25]: This was incorrect, not sure why. `deno` is only created when the client creates the stub files, but not when there is a valid schema that can be generated from.

File: platforms-serverless/serverless-framework-lambda/tsconfig.json [hunk 4]
```diff
@@ -5,6 +5,7 @@
     "outDir": ".",
     "strict": true,
     "lib": ["esnext", "dom"],
-    "esModuleInterop": true
```
Comment (millsp) [inline platforms-serverless/serverless-framework-lambda/tsconfig.json:8]: To suppress a type issue in `lodash`.

File: platforms-serverless/serverless-framework-lambda/tsconfig.json [hunk 5]
```diff
@@ -5,6 +5,7 @@
...
+    "esModuleInterop": true,
+    "skipLibCheck": true,
```
Comment (millsp) [inline platforms-serverless/serverless-framework-lambda/tsconfig.json:10]: To suppress a type issue in `lodash` itself.

**Comment by millsp** [comment]
Trying to fix Redwood, among other things, the tests are not passing because of other [reasons](https://prisma-company.slack.com/archives/CL0AJEVSA/p1693528780922799) (both fail on Redwood 3 (previous), and Redwood 6 (this PR)).

File: platforms-serverless/serverless-framework-lambda/tsconfig.json (hunk 5 above)
Comment (Jolg42) [reply platforms-serverless/serverless-framework-lambda/tsconfig.json:10]: Make this a comment then, it's ok to have a comment in tsconfig
```suggestion
    // workaround for a type issue in lodash itself
    "skipLibCheck": true,
```

File: platforms-serverless-vercel/vercel-with-redwood/api/jsconfig.json [hunk 6]
```diff
@@ -1,11 +1,11 @@
 {
   "compilerOptions": {
     "noEmit": true,
-    "allowJs": true,
     "esModuleInterop": true,
     "target": "esnext",
     "module": "esnext",
     "moduleResolution": "node",
+    "skipLibCheck": false,
```
Comment (Jolg42) [inline platforms-serverless-vercel/vercel-with-redwood/api/jsconfig.json:8]: Why is it needed for Redwood?

File: platforms-serverless-vercel/vercel-with-redwood/package.json [hunk 7]
```diff
@@ -22,8 +21,8 @@
     "root": true
   },
   "engines": {
-    "node": "16.x",
-    "yarn": ">=1.15 <2"
+    "node": "=16.x",
```
Comment (Jolg42) [inline platforms-serverless-vercel/vercel-with-redwood/package.json]: Why not
```suggestion
    "node": ">=16.x",
```

**Review by Jolg42 — APPROVED** [review]
Looks good to me! Interesting to see that something is broken with Redwood/Vercel

File: binaries/pkg/run.sh [hunk 8]
```diff
@@ -3,6 +3,7 @@
 set -eu
 
 pnpm install
+pnpm prisma generate
```
Comment (janpio) [inline binaries/pkg/run.sh:6]: Why is this needed explicitly?

File: docker/almalinux-8-amd64-openssl-1.1.x/run.sh (hunk 2 above)
Comment (janpio) [reply docker/almalinux-8-amd64-openssl-1.1.x/run.sh:33]: Why is all this needed? This should happen automatically, under the hood, right?

Comment (millsp) [reply docker/almalinux-8-amd64-openssl-1.1.x/run.sh:33]: Because our dependencies are cached to speed up the runs, we end up in a situation where the `postinstall` of Prisma Client is not always triggered, similar to what we had with Vercel.

File: platforms-serverless-vercel/vercel-with-redwood/api/jsconfig.json (hunk 6 above)
Comment (millsp) [reply platforms-serverless-vercel/vercel-with-redwood/api/jsconfig.json:8]: It was added by Redwood, not sure. Maybe for performance?

File: binaries/pkg/run.sh (hunk 8 above)
Comment (millsp) [reply binaries/pkg/run.sh:6]: Because our dependencies are cached to speed up the runs, we end up in a situation where the postinstall of Prisma Client is not always triggered, similar to what we had with Vercel.

File: platforms-serverless-vercel/vercel-with-redwood/api/jsconfig.json (hunk 6 above)
Comment (Jolg42) [reply platforms-serverless-vercel/vercel-with-redwood/api/jsconfig.json:8]: I see, and less problems from users too maybe 😄
